client/src/components/UserSignUpForm.js

```diff
@@ -78,18 +77,14 @@ export class UserSignUpForm extends Component {
   render() {
     const { email, name, password, confirmationPassword } = this.state;
     return (
-      <Grid centered column={16}>
-        <Grid.Column width={6}>
+      <Grid padded="horizontally" centered column={16}>
+        <Grid.Column computer="8" mobile="16" tablet="8" width={6}>
```

Comment:
I've played a bit with the different value, maybe this one looks better `largeScreen="6" computer="10" mobile="16" tablet="10"`. You'll need to use largeScreen otherwise it's the form is too big on big screen.

Reply:
yes , you are right, 
it is changed now